### DISCUSSION TITLE
Problems with the start button in Windows 11

### DISCUSSION THREAD

**Discussion by lordths** [valinet/ExplorerPatcher#2277] [Q&A]
Hello community, Today I reinstalled Windows 11 on my computer and when updating to its latest version I installed our beloved explorer patcher but I realized that the start button stopped working in its version of Windows 11. Like Windows 10 it works fine but it is crashing every so often.

Do you know of any solution?

**Comment by Amrsatrio** [comment]
Update to the latest prerelease.
